src/mesh/mesh.h

```diff
@@ -70,6 +70,23 @@ class BoundarySegment {
     static Input::Type::Record input_type;
 };
 
+class EquivalentMeshMap{
+public:
```

Comment:
Add a short comment on the purpose of the structure.

Reply:
done